lib/java/PlagiarismDetection/src/Tokenizer/ANTLRDynamicTokenizer.java

```diff
@@ -216,15 +215,14 @@ private TokenList processLexerOutput(ArrayList<String> lexerOutput) throws Excep
     TokenList tokens = new TokenList();
     ArrayList<String> filteredLexerOutput = new ArrayList<String>();
 
-
     // Remove Unmapped and Ignored Token names so that we can predict end of statement in the next loop
     for (String tokenString : lexerOutput ) {
       // Extract info
       String[] cols = tokenString.split(","); 
       String tokenClassString;      
 
       if (cols.length >= 4){
-	// Defensive programming
+      	// Defensive programming
```

Comment:
Mention for what type of defense?

Reply:
Previously the lexerOutput comes from std i/o via running antlr3 externally. Now antlr4 produces the lexerOutput internally. So is defensive programming still needed? (I have not touched this code segment on defensive programming from the start as I was under the impressive that this was included to err on the safe side.)

Reply:
Ok.  Got it!